### ISSUE TITLE
Allow rounding within `.agg()`

### ISSUE THREAD

**Issue by markfairbanks** [pola-rs/polars#1336]
Is it possible to allow the use of `.round()` after aggregation? Or is it recommended to use `.with_columns()` after the aggregation?

```python
import numpy as np
import polars as pl
from polars import col

test_df = pl.DataFrame(
    {'groups': ['a','a','b'],
     'vals': np.array([1.111, 2.222, 3.333])}
)

test_df.groupby('groups').agg(col('vals').mean().round(2))
```

**Current output**
```
shape: (2, 2)
┌────────┬──────────────┐
│ groups ┆ vals_mean    │
│ ---    ┆ ---          │
│ str    ┆ list [f64]   │
╞════════╪══════════════╡
│ "a"    ┆ [1.67, 3.33] │
├╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌╌╌╌┤
│ "b"    ┆ [0]          │
└────────┴──────────────┘
```

**Expected output**
```
shape: (2, 2)
┌────────┬───────────┐
│ groups ┆ vals_mean │
│ ---    ┆ ---       │
│ str    ┆ f64       │
╞════════╪═══════════╡
│ "a"    ┆ 1.67      │
├╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌╌┤
│ "b"    ┆ 3.33      │
└────────┴───────────┘
```

**Comment by ritchie46** [comment]
This is a bug, this should produce the latter dataframe. Will fix this.

**Comment by ritchie46** [comment]
Fixed in:  #1337

**Comment by ritchie46** [comment]
Fixed and being released at the moment: https://github.com/pola-rs/polars/pull/1339/checks?check_run_id=3569701202
